src/services/ai-server/ai-persona-service/ai.persona.service.interface.ts

```diff
@@ -39,4 +39,10 @@ export class IAiPersonaService extends IAuthorizable {
     description: 'The body of knowledge ID used for the AI Persona Service',
   })
   bodyOfKnowledgeID!: string;
+
+  @Field(() => Date, {
+    nullable: true,
+    description: 'When wat the body of knowledge of the VC last updated.',
+  })
+  bodyOfKnowledgeLastUpdated?: Date;
```

Comment:
doesn't this field need to be set to `undefined/null` at some point? - if not, then it's never null.

Reply:
I don't understand that completely - the question mark gives me the `undefined`. For newly created VCs before the ingestion is completed this will undefined.

Reply:
What about on re-ingestion? Shouldn't it be set to undefined?

Reply:
Do you mean to set it to undefined while the BoK is being re-ingested? Does that make sense? The old one is available while the BoK is being updated. I agree we can have a state like `UPDATING` or something, but I'd say this is outside this PR's scope and can't be handled with a single column.

Reply:
The problem is that it does not display the right behavior to the user.
Lets image this scenario:
I have just re-ingested, and my VC is already ready but has no clue about the latest knowledge I gave it, it could even not work, because it's currently initializing.